src/CMakeLists.txt

```diff
@@ -317,6 +316,47 @@ set(THIRD_PARTY_SRCS
     3rdparty/sqlite3.c
 )
 
+# Highwayhash. Highwayhash is a bit special since it has architecture dependent code...
+
+set(HH_SRCS
+    3rdparty/highwayhash/highwayhash/sip_hash.cc
```

Comment:
So - this may be a stupid question - but can you still just include the source-files like this here then? Different from libkqueue/rapidjson, highwayhash is not a cmake project - and since we don't need it anywhere else, it is kind of neat to just be able to specify the highwayhash source files directly in the src-level CMakeLists.txt.

Reply:
For this, I'd suggest to fork the repo into the `zeek` organization and then add the submodule as `aux/highwayhash` (same as like libkqueue/rapidjson... seems a bit preferable in terms of code control/maintenance), but then still fine to directly put the required compilation logic in our own `src/CMakeLists.txt` as already done (different than what libkqueue/rapidjson do, but don't see much need for consistency there -- how the external project/code is structured and how we interface with it are likely going to be vary a lot).

Reply:
Done